packages/babel-highlight/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@babel/highlight",
+  "version": "7.0.0-beta.39",
+  "description": "Syntax highlight JavaScript strings for output in terminals.",
+  "author": "Sebastian McKenzie <sebmck@gmail.com>",
```

Comment:
Shouldn't we change the author for new packages? Maybe @hzoo?

Reply:
I wasn't sure what to put. Since he was listed as the author on code-frame, I assumed most of this code was written by Sebastian, so I kept it the same since I didn't add much.

Reply:
Yes good point ok

Reply:
I guess it could even be you @suchipi? It's a new package name, not really based on who wrote the most, although even for new packages we have been doing that which I'm not sure about either?

Reply:
I'm fine changing it to my name 🤷‍♀️